### DISCUSSION TITLE
Sync after permissions update

### DISCUSSION THREAD

**Discussion by khanc** [abraunegg/onedrive#1337] [Q&A] [answered]
I changed sync_dir_permissions and sync_file_permissions. How do I make onedrive update existing permissions to the new ones. I tried --synchronize --resync but it did not work.

**Comment by abraunegg** [comment, marked as answer]
@khanc 
> I changed sync_dir_permissions and sync_file_permissions. How do I make onedrive update existing permissions to the new ones. I tried --synchronize --resync but it did not work.

Of course `--resync` will not work for that sort of action. The configuration options 'sync_dir_permissions' & 'sync_file_permissions' only change the permissions for new files that are downloaded from OneDrive, `--resync` does not re-download existing files.

To do what you want to do, you need to research `chmod` and `chown` and recursively change the permissions and ownership on the existing files and folders manually. That is out of scope for this client.

**Note:** The default permissions are the most secure for your files on your system.
